### DISCUSSION TITLE
Welcome to soundstorm-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/soundstorm-pytorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by buchylx** [comment]
Go for it, bro! This looks super impressive. Can't wait to see you nail it and achieve success!

**Comment by toddparis** [comment]
I keep getting the same fairseq error -  Could not build wheels for fairseq, which is required to install pyproject.toml-based projects.  I have updated all dependencies and reinstalled.  Any suggestions?

**Reply by xelibrion** [reply]
Try downgrading Python version - some pre-built wheels are available for older versions of Python.
